### PR TITLE
pmbootstrap: update to 1.37.0

### PR DESCRIPTION
**Pull request by stacyharper**
cant findout how to apply templates after creation but briefly:

- I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by paper42** [comment]
Please fill in the PR template

File: srcpkgs/pmbootstrap/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://postmarketos.org"
 distfiles="https://gitlab.com/postmarketOS/pmbootstrap/-/archive/${version}/pmbootstrap-${version}.tar.gz"
-checksum=d53885627382dd37536bf28edd538d10e37fdd1ccd3117b34eac35df3ec1b2e6
+checksum=38136c8600964f8ea2cdeef26293d0f39cdf73812454df1961d3b0e15c2b246a
 
 do_check() {
 	# tests require chroot
```
Comment (paper42) [inline srcpkgs/pmbootstrap/template:16]: `make_check=no # tests require chroot`

**Review by paper42 — COMMENTED** [review]
Please remove coreutils and procps-ng from dependencies. coreutils is in void base and pmbootstrap removed coreutils and procps dependency in 1.35.

File: srcpkgs/pmbootstrap/template (hunk 1 above)
Comment (stacyharper) [reply srcpkgs/pmbootstrap/template]: ya !

**Pull request merged**